components/modal/cmp/src/CookiesNotification/index.js

```diff
@@ -0,0 +1,21 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import Notification from '@s-ui/react-molecule-notification'
+
+export class CookiesNotification extends Component {
+  render() {
+    const {buttons} = this.props
+    return (
+      <Notification
```

Comment:
So it's ok to use a notification?

Reply:
We will see. Tomorrow I've a meeting and I hope to clarify this.